recipes/wirelessStatus.mdwn

```diff
@@ -0,0 +1,50 @@
+# Wireless status widget/watcher
+Get wireless Quality link (converted to percentages) and wireless status.
+
+## Requirements:
+- `iw`
+
+## Usage:
+- Download [wirelessStatus.lua](https://awesomewm.org/recipes/wirelessStatus.lua) file and put it
+  into awesome's folder (like `~/.config/awesome/widgets/wirelessStatus.lua`)
+- Add widget to `theme.lua`:
+
+  ```lua
+  local widgets = {
+      wirelessStatus = require("widgets/wirelessStatus"),
+  }
+  -- Wireless status widget
+  theme.wirelessStatus = widgets.wirelessStatus({
+      notification_preset = { font = "Mononoki Nerd Font 10", fg = theme.fg_normal },
+      timeout = 10,
+      settings = function(self)
+          if self.status == "1" or self.status == "" then
```

Comment:
I didn't find any docs for `/proc/net/wireless`, so I am not sure what its status field means. It might be worth adding a bit of explanation to this.

Reply:
According to [this](https://www.hpl.hp.com/personal/Jean_Tourrilhes/Linux/Linux.Wireless.Extensions.html) website `status` field is device dependent information. I am going to add some notes.